mopidy/http/actor.py

```diff
@@ -75,14 +75,14 @@ def on_stop(self):
         self.server.stop()
 
     def on_event(self, name, **data):
-        on_event(name, **data)
+        on_event(name, self.server.io_loop, **data)
 
 
-def on_event(name, **data):
+def on_event(name, ioloop, **data):
```

Comment:
For consistent naming:

```suggestion
def on_event(name, io_loop, **data):
```